infra/lib/api-builder.ts

```diff
@@ -118,6 +123,53 @@ export function apiBuilder(
         api: api.apiGatewayV2,
     });
 
+    //Comment Resources
+    const commentService = buildCommentService(
+        scope,
+        storageResources.dynamo.commentStorageTable,
+        storageResources.dynamo.assetStorageTable
+    );
+    attachFunctionToApi(scope, commentService, {
```

Comment:
Lets move all of these attachments to a single call function with that takes the routePath as a param

Reply:
I used a for loop like some of the other places that attach functions to the API such as [here](https://github.com/awslabs/visual-asset-management-system/blob/d14fa7a599e00b7ee4cd582bab24f1ac8eb1611f/infra/lib/api-builder.ts#L358-L364)